app/controllers/vm_common.rb

```diff
@@ -1142,6 +1142,8 @@ def replace_right_cell(options = {})
             presenter.update(:form_buttons_div, '')
             presenter.remove_paging.hide(:form_buttons_div)
           end
+        elsif @sb[:action] == 'rename'
+          presenter.hide(:form_buttons_div, '')
```

Comment:
can't it be like `presenter.hide(:form_buttons_div)` , do we need to pass empty data to it?

Reply:
I will try now